src/test/java/com/adobe/cq/commerce/graphql/client/impl/GraphqlAemContext.java

```diff
@@ -63,19 +72,61 @@ public static AemContext createContext(Map<String, String> contentPaths) {
                 props.put("configBucketNames", new String[] { "settings" });
                 serviceConfiguration.update(props);
             }).build();
+
+        // Load page structure
+        contentPaths.entrySet().iterator().forEachRemaining(entry -> {
+            ctx.load().json(entry.getValue(), entry.getKey());
+        });
+
+        ServiceUserMapped serviceUserMapped = Mockito.mock(ServiceUserMapped.class);
+        ctx.registerService(ServiceUserMapped.class, serviceUserMapped, ImmutableMap.of(ServiceUserMapped.SUBSERVICENAME,
+            "cifconfig-reader-service"));
```

Comment:
Shouldn't this be `graphql-client-configuration`? If yes, then this test has a problem because it passes although the service user is not the right one.